src/Provision.js

```diff
@@ -3,10 +3,15 @@ const fs = require("fs");
 const path = require("path");
 const { ElvUtils } = require("../src/Utils");
 const Utils = require("@eluvio/elv-client-js/src/Utils.js");
+const { ElvAccount } = require("../src/ElvAccount.js");
 
-const TYPE_LIVE_DROP_EVENT_SITE = "Eluvio LIVE Drop Event Site";
-const TYPE_LIVE_TENANT = "Eluvio LIVE Tenant";
-const TYPE_LIVE_MARKETPLACE = "Eluvio LIVE Marketplace";
+const { TenantProvisionConfig } = require("./TenantProvisionConfig.js");
+const {Config} = require("./Config");
+const t = TenantProvisionConfig.tenantProvisionConfig;
```

Comment:
Let's not make this global - that way the functions could also be used in other tools.  We can add it as an argument to all functions that need it.  Make it the first argument